### ISSUE TITLE
ReferenceError: Can't find variable: jQuery

### ISSUE THREAD

**Issue by supersonicBlonde** [giakki/uncss#268]
I have this issue when i run the following 

`uncss --htmlroot _site _site/**/**/*.html | cssnano > css/s.min.css`

Could it be easy to resolve ?

```
ReferenceError: Can't find variable: jQuery

  https://cdn.snipcart.com/scripts/2.0/snipcart.js:3
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
ReferenceError: Can't find variable: jQuery

  https://cdn.snipcart.com/scripts/2.0/snipcart.js:3
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
ReferenceError: Can't find variable: jQuery

  https://cdn.snipcart.com/scripts/2.0/snipcart.js:3
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
ReferenceError: Can't find variable: jQuery

  https://cdn.snipcart.com/scripts/2.0/snipcart.js:3
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
ReferenceError: Can't find variable: jQuery

  https://cdn.snipcart.com/scripts/2.0/snipcart.js:3
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
ReferenceError: Can't find variable: jQuery

  https://cdn.snipcart.com/scripts/2.0/snipcart.js:3
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
ReferenceError: Can't find variable: jQuery

  https://cdn.snipcart.com/scripts/2.0/snipcart.js:3
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
ReferenceError: Can't find variable: jQuery

  https://cdn.snipcart.com/scripts/2.0/snipcart.js:3
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
ReferenceError: Can't find variable: jQuery

  https://cdn.snipcart.com/scripts/2.0/snipcart.js:3
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1 in e
  https://cdn.snipcart.com/scripts/2.0/snipcart.js:1
```

**Comment by RyanZim** [comment]
Try opening the pages in a normal browser and check the developer console. If there are no errors there, this is probably a duplicate of https://github.com/giakki/uncss/issues/232 or https://github.com/giakki/uncss/issues/220.

Sometimes this error doesn't affect the css output. Check if your pages are styled correctly with the filtered css. In that case, you can ignore the "errors".

Also, if you have inline JS code, sometimes moving it to an external file helps.

I think this is a PhantomJS issue; #265 might solve this.

**Comment by RyanZim** [comment]
@supersonicBlonde Any progress on this? Can this be closed?

**Comment by RyanZim** [comment]
@mikelambert Can you close?

**Comment by LuisAlejandro** [comment]
For me, what happened was that before these `ReferenceError: Can't find variable: jQuery` errors there was another error saying that the jquery lib cound not be loaded. Looking more I found out that you removed the timeout option on 1.17.1 https://github.com/uncss/uncss/compare/0.17.1...master#diff-1cfb9e67802c00388be7d908e0bb2e6cL254 (you have a PR here https://github.com/uncss/uncss/pull/418 to revert it).

So I downgraded to version 0.17.0, set a high timeout, jQuery now loads and the error is gone.
